spec/integration/api_resource_comments_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'swagger_helper'
+
+describe ApiResourceCommentsController do
+  let(:user) { create(:user) }
+  let(:resource) { create(:resource, user:) }
+  let(:resource_id) { resource.id }
+
+  before do
+    sign_in user
+  end
+
+  path '/api/resources/{resource_id}/resource_comments' do
+    get 'Retrieves a list of resource comments' do
+      tags 'resource_comments'
```

Comment:
This tag is for the api-docs UI, I'd write it in a more "UI friendly" way, like Resource Comments

Reply:
Agree, "Resource Comments" it's more friendly